common/ferrostar/src/routing_adapters/osrm/models.rs

```diff
@@ -187,6 +189,23 @@ pub struct BannerContent {
     pub roundabout_exit_degrees: Option<u16>,
 }
 
+#[derive(Deserialize, Debug)]
+pub struct SubManeuverBannerContent {
+    pub text: String,
+    pub components: Vec<SubManeuverBannerContentComponent>,
+}
+
+#[derive(Deserialize, Debug)]
+pub struct SubManeuverBannerContentComponent {
+    #[serde(rename = "type")]
+    pub component_type: Option<String>,
+    pub directions: Option<Vec<String>>,
```

Comment:
i only focused on `directions` here, but didn't think too much about other `subs`, like just lists of `type` + `text`, or `type` + `image_base_url`, etc, as per the sample json in this file. should we handle these also?

Reply:
👍 

You know what's funny? I just realized we already have another example from Mapbox's public docs which I had forgotten about (see the `deserialize_banner_instruction` test) 😂 Let's keep both; good to have diversity of API tests in here. We could probably turn one or both of those into snapshot tests now that I think of it.